SCClassLibrary/Common/Control/Buffer.sc

```diff
@@ -370,6 +371,7 @@ Buffer {
 	}
 
 	zero { arg completionMessage;
+		if(bufnum.isNil) { Error("Cannot call % on a % that has been freed".format(thisMethod.name, this.class.name)).throw };
```

Comment:
you could simplify these. just define one method:
```
checkForSend {
    if(bufnum.isNil) { Error("Cannot call % on a % that has been freed".format(thisMethod.name, this.class.name)).throw };
}
```

and then you just add the `this.checkForSend` everywhere.